SPECS/LICENSES-AND-NOTICES/data/licenses.json

```diff
@@ -801,7 +801,8 @@
             "origin": "TreasureData",
             "license": "[ASL 2.0 License](http://www.apache.org/licenses/LICENSE-2.0)",
             "specs": [
-                "td-agent"
+                "td-agent",
+                "td-agent-bit"
```

Comment:
I'm a bit confused- you say this is an "original version for CBL-Mariner" in the spec, but it's listed here under "TreasureData". Where is this spec from?